react-carousel-component/my-app/src/Carousel.tsx

```diff
@@ -0,0 +1,65 @@
+import { BsChevronLeft } from 'react-icons/bs';
+import { BsChevronRight } from 'react-icons/bs';
+// import { BsCircleFill } from 'react-icons/bs';
+import { BsCircle } from 'react-icons/bs';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  images: string[];
+};
+
+export default function Carousel({ images }: Props) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  function handlePrevClick() {
+    setActiveIndex((activeIndex - 1 + images.length) % images.length);
+  }
+
+  function handleNextClick() {
+    setActiveIndex((activeIndex + 1 + images.length) % images.length);
```

Comment:
`+ images.length` isn't needed on this one. It's only used above to keep the left-hand side of `%` from going negative.